packages/ethash/src/index.ts

```diff
@@ -141,16 +140,26 @@ export class Miner {
   }
 }
 
+export type EthashCacheDB = Level<
```

Comment:
Is `level-mem` not compliant with the `abstract-level` API?  I see that you're typing it to `any` in a number of places and that typescript is complaining about a missing `location` property if you don't do so.

Reply:
That makes sense to me.  This is a big improvement overall so I'm fine with that for now since additional fixes would definitely not be breaking changes for us.